pkg/nfs/nodeserver.go

```diff
@@ -73,6 +74,17 @@ func (ns *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
+	if ns.Driver.perm != "" {
+		perm, err := strconv.ParseUint(ns.Driver.perm, 8, 32)
```

Comment:
Can you please convert it to uint just after parsing cmdline, so the driver exits with a clear error message? Reporting cmdline error in CSI error code looks weird.

Reply:
done, it's in cmd now.